src/utils/client.js

```diff
@@ -142,7 +142,27 @@ const transformCountyCode = (code) => {
   return code ? code.padStart(3, "0") : "county-n-a";
 };
 
+function getPropByString(obj, propString) {
+  if (!propString)
+    return obj;
+
+  var prop, props = propString.split('.');
+
+  for (var i = 0, iLen = props.length - 1; i < iLen; i++) {
+    prop = props[i];
+
+    var candidate = obj[prop];
+    if (candidate !== undefined) {
+      obj = candidate;
+    } else {
+      break;
+    }
+  }
+  return obj[props[i]];
+}
```

Comment:
Use `let/const` for `var` and the arrow notation for consistency. I still think we use `getField`, copy and paste :)